### ISSUE TITLE
homey support , all is working fine.

### ISSUE THREAD

**Issue by biemond** [rgerganov/py-air-control#23]
thanks for your hard work. 

I am made a nodejs homey app, see https://homey.app/en-us/app/com.athom.philipsair/Philips-Air/ . and it works awesome, with triggers , conditions and insights. 

homey has nodejs 8 embedded  so it was a challenge with bigint, modulo Power and AES but comparing your pip data with my local data solved all my issues.

Keep up the good work and I gave you credits on my readme.

Also how did you find out about this cloud thing. they keep this very secret somehow.

Cheer Edwin

**Comment by rgerganov** [comment]
Thanks for the feedback and the credits :)
You can read about the cloud control on my blog: https://xakcop.com/post/cloud-air-purifier/

**Comment by webosa** [comment]
Any chance of getting this to work with Smartthings .. started looking to that myself, but I am way over my head there.

**Comment by biemond** [comment]
you mean with homey?  so homey can replace the smartthings hub. so you can use triggers or conditions and make your flows to do things and combine this with the air purifier

**Comment by biemond** [comment]
@rgerganov with your decompiler action did you see something like supported options per device. 

For example for mode on AC5659_10 I only support 
```
mode_str = {'P': 'auto', 'A': 'allergen', 'M': 'manual', 'B': 'bacteria'}
```
and not sleep or night. like you are showing.
```
    mode_str = {'P': 'auto', 'A': 'allergen', 'S': 'sleep', 'M': 'manual', 'B': 'bacteria', 'N': 'night'}
```

**Comment by rgerganov** [comment]
Yes, it should be possible to detect the purifier model and then filter only the options available for it but I think this is an overkill.
